qualification/conftest.py

```diff
@@ -281,9 +285,20 @@ def matplotlib_report_style() -> Generator[None, None, None]:
         yield
 
 
+@pytest.fixture(autouse=True)
+def quiet_spead2(caplog: pytest.LogCaptureFixture) -> None:
+    """Ensure spead2 only logs warnings and bug.
```

Comment:
Warnings and bug?
Maybe "warnings and above"?

Reply:
Yes, that's exactly what was in my brain but my traitor fingers typed it wrong. Will fix.